app/src/main/res/layout/activity_group.xml

```diff
@@ -77,29 +77,32 @@
                 android:layout_width="@dimen/menu_avatar_size"
                 android:layout_height="@dimen/menu_avatar_size"
                 android:background="@drawable/circle_fill_grey"
-                android:src="@drawable/ic_face_white_48dp" />
+                android:src="@drawable/ic_face_white_48dp"/>
 
             <de.hdodenhof.circleimageview.CircleImageView
                 style="@style/MenuAvatarStyle"
                 android:layout_width="@dimen/menu_avatar_size"
                 android:layout_height="@dimen/menu_avatar_size"
                 android:background="@drawable/circle_fill_grey"
-                android:src="@drawable/ic_face_white_48dp" />
+                android:src="@drawable/ic_face_white_48dp"/>
 
             <de.hdodenhof.circleimageview.CircleImageView
                 style="@style/MenuAvatarStyle"
                 android:layout_width="@dimen/menu_avatar_size"
                 android:layout_height="@dimen/menu_avatar_size"
                 android:background="@drawable/circle_fill_grey"
-                android:src="@drawable/ic_face_white_48dp" />
+                android:src="@drawable/ic_face_white_48dp"/>
 
-            <de.hdodenhof.circleimageview.CircleImageView
+            <com.compscieddy.meetinthemiddle.FontTextView
+                style="@style/ButtonOutlineAccent"
                 android:id="@+id/invite_button"
-                style="@style/MenuAvatarStyle"
-                android:layout_width="@dimen/menu_avatar_size"
-                android:layout_height="@dimen/menu_avatar_size"
-                android:background="@drawable/circle_fill_grey"
-                android:src="@drawable/ic_plus_white_48dp" />
+                android:layout_width="wrap_content"
+                android:layout_height="match_parent"
+                android:layout_marginLeft="5dp"
+                android:gravity="center_vertical"
```

Comment:
gravity and padding should go in the style as well I think. ok maybe not padding actually, but the gravity